File: Perl/MailSender.pl
#!/usr/bin/perl

use Net::SMTP::SSL;

use strict;
use warnings;

our $smtp = Net::SMTP::SSL->new("smtp.qq.com", Port => 465) or die "Failed to connect to the SMTP server!";

sub addAuthor {
	$smtp->auth('961950442@qq.com', 'Seven1001');
	$smtp->mail('961950442@qq.com');
	$smtp->to('2307849251@qq.com', '961950442@qq.com');
}

sub sendMail {
	#Start the mail
	$smtp->data();

	#Edit the header
	$smtp->datasend("To: 2307849251\@qq.com\n");
	$smtp->datasend("From: 961950442\@qq.com\n");
	$smtp->datasend("Subject: Send from perl to test.\n");

	#Separate the header from the body
	$smtp->datasend("\n");

	# Send the body.
	$smtp->datasend("I love you, indigo!\n");
	$smtp->datasend("Seven...\n");

	# Finish sending the mail
	$smtp->dataend(""); 

	# Close the SMTP connection
	$smtp->quit();
	
	# $?-- is for the command status outside the perl compiler.
	# so don't return the $?.
}

#print addAuthor(), "\n";
#print sendMail(), "\n";

if(addAuthor() && sendMail()) {
	print "Email is sending...\n";
}
else {
	print "Error in the last step";
}
	

File: Python/MailSender.py
#!/usr/bin/python3.4
# Send an email!

from smtplib import SMTP_SSL
from email.parser import HeaderParser

# If the e-mail headers are in a file, uncomment these two lines:
# with open(messagefile) as fp:
#     headers = Parser().parse(fp)

# This is an email.message.Message!
# Or for parsing headers in a string, use:
headers = HeaderParser().parsestr("From: 961950442@qq.com\n"
				"To: 2307849251@qq.com\n"
        		"Subject: Test message\n"
        		"\n"
        		"I love you, you know!\n"
        		"----python..\n")
# print(type(headers))
# Just list the process.
try:
	conn = SMTP_SSL('smtp.qq.com', port=465)
	conn.login("961950442@qq.com", "Seven1001")
	tolist = ["2307849251@qq.com"]
	try:
		conn.sendmail('961950442@qq.com', tolist, headers.as_string())
	finally:
		conn.quit()
except Exception:
	print("Catch Exception!")
else:
	print("The mail is sending!")